CONTRIBUTING.md

```diff
@@ -94,22 +98,26 @@ Make sure `yarn lint`, `yarn test` and `yarn vscode:prepublish` all work.
 **2. integration tests**
 
 There are two debugging launch configurations defined in `.vscode/launch.json`:
-  * Debug Tests
-  * Launch Extension
+
+- Debug Tests
+- Launch Extension
 
 To debug the extension, [change the launch configuration](https://code.visualstudio.com/docs/editor/debugging#_launch-configurations) to **Launch Extension** and start debugging.
 
 **3. eat our own dog food**
 
 The ultimate test is to actually use it in our real day-to-day working environment for a while. There are multiple ways to do this:
+
 - by command line: `code --extensionDevelopmentPath=your-local-vscode-jest`
 - by environment variable: `CODE_EXTENSIONS_PATH`
 - by symlink:
 
   Here is a mac example:
+
   ```
   $ cd ~/.vscode/extensions
-  $ mv Orta.vscode-jest-2.7.0 vscode.orig
-  $ ln -s Orta.vscode-jest-2.7.0 your-local-vscode-jest
+  $ mv orta.vscode-jest-4.0.0 vscode.orig
+  $ ln -s your-local-vscode-jest orta.vscode-jest-4.0.0
```

Comment:
This section is the only thing I actually changed.  What matters is that `orta` is lowercase and the arguments to `ln` are in the right order.